homeassistant/components/dnsip/config_flow.py

```diff
@@ -102,8 +116,12 @@ async def async_step_user(
             name = DEFAULT_NAME if hostname == DEFAULT_HOSTNAME else hostname
             resolver = user_input.get(CONF_RESOLVER, DEFAULT_RESOLVER)
             resolver_ipv6 = user_input.get(CONF_RESOLVER_IPV6, DEFAULT_RESOLVER_IPV6)
+            port = user_input.get(CONF_PORT, DEFAULT_PORT)
+            port_ipv6 = user_input.get(CONF_PORT_IPV6, DEFAULT_PORT_IPV6)
```

Comment:
As there is a default in the schema there will always be something here so no need to use `.get()` to specify a default value

Reply:
Also below

Reply:
I will make the same change to the two lines above as well

Reply:
Removing the .get() defaults 
```python
            resolver = user_input.get(CONF_RESOLVER)
            resolver_ipv6 = user_input.get(CONF_RESOLVER_IPV6)
            port = user_input.get(CONF_PORT)
            port_ipv6 = user_input.get(CONF_PORT_IPV6)
```
Causes test_form to fail
```
FAILED tests/components/dnsip/test_config_flow.py::test_form - AssertionError: assert {'resolver': None, 'port': None, 'resolver_ipv6': None, 'port_ipv6': None} == {'resolver': '208.67.222.222', 'resolver_ipv6': '2620:119:53::53', 'port': 53, 'port_ipv6': 53}
  
  Differing items:
  {'resolver': None} != {'resolver': '208.67.222.222'}
  {'port_ipv6': None} != {'port_ipv6': 53}
  {'port': None} != {'port': 53}
  {'resolver_ipv6': None} != {'resolver_ipv6': '2620:119:53::53'}
```